flake.lock

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Is this a strictly-generated file? If so, it might be good to add a `.gitattributes` file and mark it as such: https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github